server/aws-lsp-partiql/build-scripts/compile-inline-tree-sitter-parser-wasm.mts

```diff
@@ -0,0 +1,13 @@
+import { readFileSync, writeFileSync } from 'fs'
+
+// Read .wasm binary file, NOTICE and LICENSE file and add description of our modifications.
+const base64EncodedBinary = readFileSync('./src/server/tree-sitter-parser/tree-sitter-partiql.wasm', 'base64')
+const modifications =
+    '`base64EncodedBinary` contains a base64 encoding of the tree-sitter-partiql parser compiled to WebAssembly.\n\tModifications around it are to import it from a JS/TS file and instantiate it using `WebAssembly.instantiate.`'
+
+// Copy the encoded binary, notice and license into a .ts file.
+const encodedPartiQLServer = `/*\n${modifications}\n*/\n\nconst base64String = atob("${base64EncodedBinary}");\nconst base64Buffer = new Uint8Array(base64String.length);\nfor (var i = 0; i < base64String.length; i++) base64Buffer[i] = base64String.charCodeAt(i);\nexport default base64Buffer;`
+writeFileSync('./src/server/tree-sitter-parser/tree-sitter-parser-inline.ts', encodedPartiQLServer)
+
+// eslint-disable-next-line no-undef
```

Comment:
Don't think we need this eslint disable here?

Reply:
We use `require()` for `fs` and `path` here because they’re specific to Node.js, where `import` isn't as compatible. Though, the code compiles and runs correctly without this eslint disable here. I've disabled the ESLint rule `@typescript-eslint/no-var-requires` here to avoid errors reported.